src/OsmMom.sol

```diff
@@ -25,42 +27,36 @@ interface AuthorityLike {
     function canCall(address src, address dst, bytes4 sig) external view returns (bool);
 }
 
+interface VatLike {
+    function file(bytes32 ilk, bytes32 what, uint256 data) external;
+}
+
+interface AutoLineLike {
+    function remIlk(bytes32 ilk) external;
+}
+
 contract OsmMom {
-    event LogNote(
-        bytes4   indexed  sig,
-        address  indexed  usr,
-        bytes32  indexed  arg1,
-        bytes32  indexed  arg2,
-        bytes             data
-    ) anonymous;
-
-    modifier note {
+    address public owner;
+    address public authority;
+    address public autoLine;
```

Comment:
Consider locating this right before or after `osms` (and possibly move `osms` here to group storage together).

Reply:
Done!